Trust: Update documentation on Portugal (#161)
pretix/pretix-docs: docs/trust/fiscal/portugal.md

### E-invoicing

All invoices in Portugal must be created by [certified invoicing software](https://vat-one-stop-shop.ec.europa.eu/national-vat-rules/portugal-vat-rules_en) and submitted electronically to the tax authority (Autoridade Tributária e Aduaneira, AT).

<!-- md:hosted -->

> On pretix Hosted, sending invoices to AT is supported through a partner.

Reviewer: ```suggestion
pretix Hosted supports sending invoices to AT through our partner Invopop S.L. and their subprocessor RUPEAL.
This feature is not available on pretix Enterprise and pretix Community.
```

> To get started, go to :navpath:Your organizer → :fa3-wrench: Settings → Plugins: and enable the plugin "E-invoicing for Portugal (via Invopop)".

Reviewer: ```suggestion
In order to set up the transmission of invoices to the Portuguese tax authority, navigate to :navpath:Your organizer → :fa3-wrench: Settings → Plugins:. 
Open the :btn:Integrations: tab. 
The list on this page includes the  "E-invoicing for Portugal (via Invopop)" plugin. 
Click the :btn:Enable: button next to it.

In the list labeled "Events with active plugin", check the events for which you want to submit invoices to AT.
Alternatively, navigate to :navpath:Your event → :fa3-wrench: Settings → Plugins: and enable the plugin "E-invoicing for Portugal (via Invopop)" for your event.
```

> Then, go to :navpath:Your organizer → :fa3-wrench: Settings → E-invoicing (PT):, fill out details about your company and follow the displayed steps for obtaining AT credentials.

Reviewer: ```suggestion
Navigate to :navpath:Your organizer → :fa3-wrench: Settings → E-invoicing (PT):, fill out details about your company and follow the displayed steps for obtaining AT credentials.
```

> Once you have completed this setup, you can go to :navpath:Your event → :fa3-wrench: Settings → Plugins: and enable the plugin "E-invoicing for Portugal (via Invopop)" for your event.

Reviewer: Remove this line if you apply suggestion https://github.com/pretix/pretix-docs/pull/161/changes#r3518843032

Author: ```suggestion
```

> Then, to :navpath:Your event → :fa3-wrench: Settings → E-invoicing: fill out the settings and follow all recommendations.

Reviewer: ```suggestion
Navigate to :navpath:Your event → :fa3-wrench: Settings → E-invoicing:, fill out the mandatory information and follow the recommendations on this page. 
```

> Invoices will be transmitted to SdI automatically through our partner Invopop S.L. and their subprocessor RUPEAL.

Reviewer: ```suggestion
pretix will automatically transmit invoices to AT through our partner Invopop S.L. and their subprocessor RUPEAL.
```

> An additional price per invoice is charged according to our [price list](https://pretix.eu/about/en/pricing).

Reviewer: ```suggestion
We will charge an additional fee per invoice according to our [price list](https://pretix.eu/about/en/pricing).
```

> pretix Enterprise and pretix Community currently do not support the Portuguese invoicing process.

Reviewer: Remove if you applied the first suggestion https://github.com/pretix/pretix-docs/pull/161/changes#r3518625318

Author: ```suggestion
```
